src/mono/sample/wasm/browser-bench/frame-main.js

```diff
@@ -30,24 +30,24 @@ try {
         mute = true;
     }
 
-    const runtime = await createDotnetRuntime({
-        disableDotnet6Compatibility: true,
-        configSrc: "./mono-config.json",
-        printErr: function () {
-            if (!mute) {
-                console.error(...arguments);
-            }
-        },
-        onConfigLoaded: (config) => {
-            if (window.parent != window) {
-                window.parent.resolveAppStartEvent("onConfigLoaded");
-            }
-            // config.diagnosticTracing = true;
-        },
-        onAbort: (error) => {
-            wasm_exit(1, error);
-        },
-    });
+    const runtime = await dotnet
+        .withModuleConfig({
+            printErr: function () {
```

Comment:
I think that the default handler we have for `printErr` is better, same for `onAbort`. You could remove it here.